debian/control

```diff
@@ -10,7 +10,6 @@ Uploaders: Jeremy Bicha <jbicha@debian.org>
 Build-Depends: debhelper-compat (= 12),
                dh-sequence-gnome,
                geoclue-2.0 (>= 2.3.1),
-               iso-codes,
```

Comment:
I think this should be submitted to the Debian packaging.

(On the other hand, I am pretty sure that, at runtime, gnome-initial-setup relies on iso-codes being installed because libgnome-desktop's `gnome-languages.c` reads `/usr/share/xml/iso-codes/iso_3166.xml`; weirdly, the `libgnome-desktop-3-17` package doesn't have this dependency.)

Reply:
This seems to have been added in https://salsa.debian.org/gnome-team/gnome-initial-setup/commit/85732c17ee37e93b568a933cac5589ce853d596c at a time when GDM was a build dependency, as far as I can tell from the commit message?

Reply:
Submitted upstream: https://salsa.debian.org/gnome-team/gnome-initial-setup/merge_requests/8